src/services/ansibleConfig.ts

```diff
@@ -29,6 +29,15 @@ export class AnsibleConfig {
         settings
       );
 
+      // TODO: remove this once we implement https://github.com/ansible/vscode-ansible/issues/312
+      // Temporary measure for displaying info that may be used for
+      // debugging execution failures for extenal commands
+      const debugInfoResult = await commandRunner.runCommand(
```

Comment:
This can be moved into utils file and called from other services initialize method as well.